db/memtable.cc

```diff
@@ -1452,18 +1452,24 @@ void MemTable::MultiGet(const ReadOptions& read_options, MultiGetRange* range,
     }
     SequenceNumber dummy_seq;
     GetFromTable(*(iter->lkey), iter->max_covering_tombstone_seq, true,
-                 callback, &iter->is_blob_index, iter->value->GetSelf(),
-                 /*columns=*/nullptr, iter->timestamp, iter->s,
-                 &(iter->merge_context), &dummy_seq, &found_final_value,
-                 &merge_in_progress);
+                 callback, &iter->is_blob_index,
+                 iter->value ? iter->value->GetSelf() : nullptr, iter->columns,
```

Comment:
IIUC, iter->value will be nullptr in case of Wide Column.

Reply:
Right, if `MultiGet` is called, `value` will be non-null and `columns` will be null in the `KeyContext`. If `MultiGetEntity` is called, it will be the other way around (`value` is null and `columns` is non-null).